src/Command/RunCommand.php

```diff
@@ -229,6 +230,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                 $job->getEncryptorFactory(),
                 $clientWrapper,
                 $loggerService,
+                new OutputFilter(60000),
```

Comment:
Do I see a default value here? 😄

Reply:
It's the only place it's set to this value, for a default i'd require at least two occurrences 😆